sdk/cosmos/azure-cosmos/azure/cosmos/http_constants.py

```diff
@@ -186,6 +191,7 @@ class HttpHeaders(object):
     EnableCrossPartitionQuery = "x-ms-documentdb-query-enablecrosspartition"
     PartitionKeyRangeID = "x-ms-documentdb-partitionkeyrangeid"
     PartitionKeyDeletePending = "x-ms-cosmos-is-partition-key-delete-pending"
+    PhysicalPartitionID = "x-ms-cosmos-physical-partition-id"
```

Comment:
where this header will be used?

Reply:
Probably left-over? At least I have never seen this header name before.

Reply:
yep my bad, this was from the previous bulk PR, will go ahead and remove it